hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -317,8 +318,7 @@ private boolean getFilesForArchive(DBCheckpoint checkpoint,
 
     // Get the snapshot files.
     Set<Path> snapshotPaths = waitForSnapshotDirs(checkpoint);
-    Path snapshotDir = Paths.get(OMStorage.getOmDbDir(getConf()).toString(),
-        OM_SNAPSHOT_DIR);
+    Path snapshotDir = Paths.get(getSnapshotsParentDir()).getParent();
```

Comment:
@hemantk-12 , Why the `parentDir` of the `getSnapshotsParentDir` is the `snapshotDir`?  Is it a typo?

Reply:
Yes, it is the naming issue in `getSnapshotsParentDir`.
I renamed and changed the function to return `snapshotDir` path directly.
I also added a comment explaining it.